snap-geotiff/src/main/java/org/esa/snap/dataio/geotiff/GeoTiffMatrixMultiLevelSource.java

```diff
@@ -70,4 +79,24 @@ public synchronized void reset() {
             }
         }
     }
+
+    @Override
+    public boolean isGlobalShifted180() {
+        return false;
+    }
+
+    @Override
+    public int getBandIndex() {
+        return this.bandIndex;
+    }
+
+    public ImageLayout buildMultiLevelImageLayout() {
```

Comment:
this method is not used any more

Reply:
The method 'buildMultiLevelImageLayout' is used the product reader classes: SpotDimapProductReader, WorldView2ProductReader, WorldView2ESAProductReader of the s2tbx project.

Reply:
okay, I had not checked out s2tbx branch.